project_task_materials/views/project_view.xml

```diff
@@ -1,25 +1,22 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
-  <data>
+<odoo>
 
     <record id="view_project_task_form_materials" model="ir.ui.view">
-      <field name="name">project.task.form.materials</field>
-      <field name="model">project.task</field>
-      <field name="inherit_id" ref="project.view_task_form2"/>
-      <field name="arch" type="xml">
-
-        <field position="after" name="work_ids">
-          <!--Materials used-->
-          <field name="material_ids" groups="project.group_tasks_work_on_tasks">
-            <tree string="Materials used" editable="top">
-              <field name="product_id"/>
-              <field name="quantity"/>
-            </tree>
-          </field>
+        <field name="name">project.task.form.materials</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project_timesheet.view_task_form2_inherited"/>
+        <field name="arch" type="xml">
+            <xpath expr="//notebook/page[@groups='project.group_tasks_work_on_tasks,project.group_time_work_estimation_tasks']" position="after">
```

Comment:
What you can do is to get a field that is inside that page, and use `ancestor`. Example: https://github.com/OCA/delivery-carrier/pull/113/files#diff-66ac07ed3b099c94da0b3e7d3a01ed85R16